src/controller/python/chip/ChipStack.py

```diff
@@ -120,10 +119,37 @@ def formatTime(self, record, datefmt=None):
             timestampStr = "%s.%03ld" % (timestampStr, timestampUS / 1000)
         return timestampStr
 
+class CHIPAsyncCallableHandle:
```

Comment:
nit: skip the "CHIP" prefix? It will be less work to do when/if we decide to switch all classes to "matter" ;)

Reply:
resolved